bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -320,6 +320,11 @@ if args.cluster_window and len(data['stat']) > 0:
                                data['timeslide_id'], args.timeslide_interval,
                                args.cluster_window)
 
+ifolist = ''
+for ifo in detectors:
+    ifolist += '%s ' % ifo
+ifolist = ifolist.rstrip()
```

Comment:
```
ifolist = ''.join(detectors)
```
would make this a one-liner, I think.

Reply:
... and as this is only used below it could be:
```
f.attrs['ifos'] = ''.join(detectors)
```

Reply:
Wait, must be reproducible and predictable, so sort detectors:
```
f.attrs['ifos'] = ''.join(sorted(detectors))
```

Reply:
Changed, though need to add them together by a space. (Needed to be able to remove triggers in the statmap code)